obfx_modules/companion-legacy/inc/hestia/common-functions.php

```diff
@@ -201,3 +209,34 @@ function themeisle_hestia_create_menu( $type ) {
 	}
 	return '';
 }
+
+/**
+ * Generate new unique widget name.
+ *
+ * @param string $widget_name Widget name.
+ *
+ * @return string
+ */
+function themeisle_hestia_get_new_widget_name( $widget_name ) {
```

Comment:
@cristian-ungureanu this is an example where I think you can improve :) the comment of the function says that the function is generating a unique name for the widget, while the function name is get_new_widget_name, which it doesn't match what the function is doing at all. By changing the name of the function to something like themeisle_hestia_generate_unique_widget_name I think it's much easier for someone else to understand what's happening in the function. Something similar for themeisle_hestia_fix_duplicate_widgets, where the comment is saying something about checking if there are duplicates, but the function name is saying something about fixing. It's not easy to understand what happens there, just some checking or some fixing too. I think that paying attention to such small details can help us all 👍